docs/onnxruntime_dependencies.dot

```diff
@@ -0,0 +1,88 @@
+digraph "GG" {
```

Comment:
What about dependencies on OpenMP?

Reply:
I didn't list all the dependencies here. For example, eigen isn't in the graph.  Yes I could put them there. But we need to find a balance between comprehensive and readable.